test/mocha/10-api.js

```diff
@@ -5,13 +5,14 @@
 
 const bedrock = require('bedrock');
 const {config} = bedrock;
-const https = require('https');
 const helpers = require('./helpers');
 const mockData = require('./mock.data');
 const {httpClient} = require('@digitalbazaar/http-client');
 const brAccount = require('bedrock-account');
 const brAuthnToken = require('bedrock-authn-token');
 const {authenticator} = require('otplib');
+const brHttpsAgent = require('bedrock-https-agent');
+const {agent} = brHttpsAgent;
```

Comment:
I want to be sure that you understand that extracting the `agent` in the header like this only works in a test spec because of how bedrock bootstraps itself and order of operation.  It is not safe to do this in normal library code.  Because we don't want people thinking it's OK to do this everywhere, I think it's best to extract the `agent` immediately prior to its use in the tests below.

Reply:
Updated.